tests/integration/cartography/intel/kubernetes/test_ingresses.py

```diff
@@ -0,0 +1,38 @@
+from cartography.intel.kubernetes.ingresses import load_ingresses
+from tests.data.kubernetes.ingresses import GET_INGRESSES_DATA
+from tests.integration.cartography.intel.kubernetes.test_services import test_load_services
+
+
+TEST_UPDATE_TAG = 123456789
+
+
+def test_load_ingresses(neo4j_session):
```

Comment:
For the tests, also query for the relationships. This will make us a bit more confident about the test data.

Reply:
Done, thanks for your feedback!